```
=========================== short test summary info ============================
FAILED api/tests/unit_tests/services/test_dataset_service_document.py::TestDocumentServiceSaveDocumentWithDatasetId::test_save_document_with_dataset_id_notion_import_truncates_names_and_cleans_removed_pages - AssertionError: expected call not found.
Expected: DocumentIndexingTaskProxy('tenant-1', 'dataset-1', ['doc-new'])
  Actual: DocumentIndexingTaskProxy('tenant-1', 'dataset-1', ['e300a539-e550-4b68-8dc5-67cd5de7cd93'])

pytest introspection follows:

Args:
assert ('tenant-1', ...7cd5de7cd93']) == ('tenant-1', ..., ['doc-new'])
  
  At index 2 diff: ['e300a539-e550-4b68-8dc5-67cd5de7cd93'] != ['doc-new']
  
  Full diff:
    (
        'tenant-1',
        'dataset-1',
        [
  -         'doc-new',
  +         'e300a539-e550-4b68-8dc5-67cd5de7cd93',
        ],
    )
===== 1 failed, 14279 passed, 4 skipped, 550 warnings in 332.12s (0:05:32) =====
```

test failed

--- a/api/tests/unit_tests/services/test_dataset_service_document.py
+++ b/api/tests/unit_tests/services/test_dataset_service_document.py
@@ -1022,6 +1022,7 @@ class TestDocumentServiceSaveDocumentWithDatasetId:
             patch.object(DocumentService, "build_document", return_value=created_document) as build_document,
             patch("services.dataset_service.clean_notion_document_task") as clean_task,
             patch("services.dataset_service.DocumentIndexingTaskProxy") as document_proxy_cls,
+            patch("services.dataset_service.uuid.uuid4", return_value="doc-new"),
         ):
             mock_redis.lock.return_value = _make_lock_context()
             session.scalars.return_value.all.return_value = [existing_keep, existing_remove]

